app/javascript/components/common/Reputation.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Icon } from './Icon'
+
+export const Reputation = ({
+  value,
+  type = 'common',
+}: {
+  value: string
+  type?: 'primary' | 'common'
```

Comment:
When is something primary reputation and when common?

Reply:
It's whether it's purple background (eg in the header) or white background. It's actually also a bit more complicated than that because originally the "primary" one was supposed to have notifications etc. It all needs renaming by me :)

Reply:
Okay. I found the names somewhat non-intuitive :)

Reply:
Yep!